digicampipe/utils/tests/test_camera.py

```diff
@@ -0,0 +1,30 @@
+'''
+testing digicam.utils.Camera here.
+Not all features it inherits ... just the two new features are tested.
+
+'''
+
+
```

Comment:
There is definitely a need for more tests regarding `cts.Camera` but this does not belong here !